pages/osx/whence.md

```diff
@@ -0,0 +1,19 @@
+# whence
+
+> A zsh builtin to indicate how a given command would be interpreted.
+
+- Interpret {{command}}, with expansion if defined as an `alias` (similar to `command -v` builtin):
```

Comment:
Perhaps this will read better?
```suggestion
- Interpret {{command}}, with expansion if defined as an `alias` (similar to the `command -v` builtin):
```